packages/react-components/source/scss/library/components/_tag.scss

```diff
@@ -0,0 +1,23 @@
+.rc-tag {
+  background: $puppet-b500;
+  border-radius: $puppet-common-border-radius;
+  display: flex;
+  width: max-content;
+
+  .rc-icon {
+    height: 12px !important;
+    width: 12px !important;
+  }
+}
+
+.rc-tag-label-background {
+  align-items: center;
+  display: flex;
+
+  .rc-tag-text {
+    color: $puppet-white;
+    height: fit-content;
+    padding-left: 8px;
+    padding-right: 4px;
+  }
```

Comment:
It looks like the spec in Figma uses a font of Calibre, 14px, bold (like h5) instead of the default Open Sans font. This obviously isn't a header though, so I'd like to confirm the typography with @Sigler. There's a precedent to using Calibre in non-headers with form labels, but if we're going to continue to do things like that, we need a simple way of defining and using those typographic styles.

Reply:
It's the same font as font as button, so let's just add `@include puppet-type-button()` like in [_button.scss](https://github.com/puppetlabs/design-system/blob/master/packages/react-components/source/scss/library/components/_button.scss#L166).